src/components/todo-item/Index.module.css

```diff
@@ -0,0 +1,91 @@
+.container {
+  padding: 8px, 16px, 8px, 16px;
+  gap: 12px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+  width: 440px;
+  height: 48px;
+  line-height: 16px;
+  font-size: 14px;
+  color: var(--primary-colour);
+  border-radius: 10px;
+  transition: box-shadow 0.2s 0s ease-out;
+}
+
+.container:hover {
+  background-color: var(--background-color);
+}
+
+.container:hover .deleteButton {
+  display: block;
+}
+
+.container.isEditing {
+  background: #fff;
+  box-shadow: 0px 0px 0px 1px rgba(0, 0, 0, 0.05),
+    0px 1px 3px 0px rgba(221, 221, 221, 1);
+}
+
+.completed {
+  text-decoration: line-through;
+}
+
+.title {
+  line-height: 16px;
+  font-size: 14px;
+  flex: 1;
+  max-width: 332px;
+  border: none;
+  outline: none;
+  font-family: var(--font-family);
+  padding: 0px;
+}
+
+.buttonTitle {
+  flex: 1;
+  max-width: 332px;
+  border: none;
+  align-items: center;
+  line-height: 16px;
+  font-size: 14px;
+  display: flex;
+  font-family: var(--font-family);
+  padding: 0px;
+  background-color: var(--primary-colour);
+  height: 100%;
+}
+
+/* deleteButton */
+.deleteButton {
+  width: 32px;
+  height: 32px;
+  border-radius: 8px;
+  border: 1px;
+  cursor: pointer;
+
+  background-size: 50%;
+
+  background-color: #ffffff;
+  border-color: #ffffff;
+  box-shadow: 0px 1px 3px 0px #00000014;
+  transition: transform 0.3s ease;
+  display: none;
+}
+
+.deleteButton:hover {
+  cursor: pointer;
```

Comment:
`.deleteButton` is used on a button element, which has `cursor: pointer` by default. You can remove this line.